checkers.kv

```diff
@@ -0,0 +1,64 @@
+<TitleMenu>:
+    BoxLayout:
+        orientation: 'vertical'
+        Label:
+            text: 'Checkers'
+        Button:
+            text: 'Settings'
+            on_release: root.manager.current = 'settings'
+        Button:
+            text: 'Play'
```

Comment:
Now that we have the screen, might as well link this to settings as mentioned above